README.md

```diff
@@ -223,6 +224,7 @@ Each subcommand generally accepts the same options and syntax, and they mirror t
 * `--templateName NAME` or `-n NAME`: define the Stencil template to use (by name, see [here for more info](Documentation/templates)) to generate the output.
 * `--templatePath PATH` or `-p PATH`: define the Stencil template to use, using a full path.
 * Note: you should specify one and only one template when invoking SwiftGen. You have to use either `-t` or `-p` but should not use both at the same time (it wouldn't make sense anyway and you'll get an error if you try)
+* `--filter REGEX` or `-f REGEX`: the filter to apply to each input path. Each command has a default filter that you can override with this option.
```

Comment:
Not sure it's worth it, and people are supposed to know how RegExes work so it's not our job to explain it to them, but given in those filters it will be pretty common to filter by extension, it could be interesting here to just add a quick reminder sentence about "e.g. use `.*` to match any characters, and don't forget to escape the dot (`\.`) if you want to match a literal dot like for an extension. For example, use `.*\.xib` to match files with a `.xib` extension."

This could be added as a `* Note:` similarly to what we have for `--templateName` vs `--templatePath` just above